File: app/src/main/java/com/feedbacktower/utilities/videotrimmer/interfaces/OnRangeSeekBarListener.java
package com.feedbacktower.utilities.videotrimmer.interfaces;


import com.feedbacktower.utilities.videotrimmer.view.RangeSeekBarView;

/**
 * Created by Deep Patel
 * (Sr. Android Developer)
 * on 6/4/2018
 */
public interface OnRangeSeekBarListener {
    void onCreate(RangeSeekBarView rangeSeekBarView, int index, float value);

    void onSeek(RangeSeekBarView rangeSeekBarView, int index, float value);

    void onSeekStart(RangeSeekBarView rangeSeekBarView, int index, float value);

    void onSeekStop(RangeSeekBarView rangeSeekBarView, int index, float value);
}

File: app/src/main/java/com/feedbacktower/utilities/videotrimmer_kt/interfaces/OnRangeSeekBarListener.kt
package com.feedbacktower.utilities.videotrimmer_kt.interfaces

import com.feedbacktower.utilities.videotrimmer_kt.view.RangeSeekBarView


interface OnRangeSeekBarListener {
    fun onCreate(rangeSeekBarView: RangeSeekBarView, index: Int, value: Float)

    fun onSeek(rangeSeekBarView: RangeSeekBarView, index: Int, value: Float)

    fun onSeekStart(rangeSeekBarView: RangeSeekBarView, index: Int, value: Float)

    fun onSeekStop(rangeSeekBarView: RangeSeekBarView, index: Int, value: Float)
}
